packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -1175,6 +1222,159 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     }
   }
 
+  /**
+   * Gets the text of the highlighed people and writes it to the clipboard
+   */
+  private writeHighlightedText() {
+    const copyText = [];
+    for (let i = 0; i < this._highlightedUsers.length; i++) {
+      const element: any = this._highlightedUsers[i];
+      const _personDetails = element._personDetails;
+      const { id, displayName, email, userPrincipalName, scoredEmailAddresses } = _personDetails;
+      let emailAddress: string;
+      if (scoredEmailAddresses && scoredEmailAddresses.length > 0) {
+        emailAddress = scoredEmailAddresses.pop().address;
+      } else {
+        emailAddress = userPrincipalName || email;
+      }
+
+      copyText.push({ id, displayName, email: emailAddress });
+    }
+    let copiedTextStr: string = '';
+    if (copyText.length > 0) {
+      copiedTextStr = JSON.stringify(copyText);
+    }
+
+    navigator.clipboard
+      .writeText(copiedTextStr)
+      .then(() => {})
+      .catch(err => {});
+  }
+
+  /**
+   * Handles the cut event when it is fired
+   */
+  private handleCut() {
+    this.writeHighlightedText();
+    this.clearState();
```

Comment:
This is removing all people from the people picker, not just the ones that are selected